tutorials/modules/ROOT/pages/create-materialized-view-from-kafka.adoc

```diff
@@ -0,0 +1,355 @@
+The result is an empty table. You don't see any results because, by default, Confluent Cloud consumers read messages, starting from the latest offset. The trades that you published already happened, and so they are not included.
++
+```
++------------+----------------------+-------------------+
+|ticker      |           price_usd  |          amount   |
++------------+----------------------+-------------------+
+```
+
+. Stop the streaming query by pressing kbd:[Ctrl + C] to close the connection to the SQL prompt.
+
+. Back in the SQL browser in Management Center, create the mapping to the topic again, but this time, add the `'auto.offset.reset'='earliest'` configuration. This configuration tells the Kafka consumer to read all data in the topic from the beginning, not just the latest.
++
+[source,sql]
+----
+-- Create a mapping to a Kafka topic called 'trades'.
+CREATE OR REPLACE MAPPING trades (
+  id BIGINT,
+  ticker VARCHAR,
+  price_usd DECIMAL,
+  amount BIGINT)
+TYPE Kafka
+OPTIONS (
+  -- Serialization format
+  'valueFormat' = 'json-flat',
+  -- Required connection configs for Kafka producer, consumer, and admin
+  'bootstrap.servers'='<YOUR BOOTSTRAP SERVER>',
+  'security.protocol'='SASL_SSL',
+  'sasl.jaas.config'='org.apache.kafka.common.security.plain.PlainLoginModule 
+  required username="<YOUR API KEY>" 
+  password="<YOUR API SECRET>";',
+  'sasl.mechanism'='PLAIN',
+  --Required for correctness in Apache Kafka clients prior to 2.6
+  'client.dns.lookup'='use_all_dns_ips',
+  -- Best practice for higher availability in Apache Kafka clients prior to 3.0
+  'session.timeout.ms'='45000',
+  'auto.offset.reset'='earliest'
+);
+----
++
+TIP: You can find your previous mapping query in the *History* tab of the SQL browser.
+
+. In the SQL prompt, enter the same streaming query that gave no results last time.
++
+[source,sql]
+----
+SELECT ticker, price_usd, amount
+  FROM trades
+  WHERE price_usd * amount > 100;
+----
++
+You should see that Hazelcast executed the query and filtered the results, using your previous trades:
++
+```
++-----------------+----------------------+-------------------+
+|ticker           |       price_usd      |       amount      |
++-----------------+----------------------+-------------------+
+|EORG             |                  14  |               20  |
+```
+
+. Stop the streaming query by pressing kbd:[Ctrl + C] to close the connection to the SQL prompt.
+
+== Step 3. Enrich the Data in the Kafka Messages
+
+Kafka messages are often small and contain minimal data to reduce network latency. For example, the `trades` topic does not contain any information about the company that's associated with a given ticker. To get deeper insights from data in Kafka topics, you can join query results with contextual data.
+
+. Open the SQL browser in Management Center.
+
+. Create a mapping to a new map in Hazelcast, in which to store the company information that you can use to enrich results from the `trades` topic.
```

Comment:
I'd split this down a bit more....something like:

The new map is for storing the company information that you'll use to enrich results from the `trades` topic.